### PR TITLE
#165054398 Setup Eslint

### PR DESCRIPTION
**Pull request by YvesIraguha**
#### What does this PR do?
- Enable eslint for linting. 
#### Description of Task to be completed?
- Enable eslint 
- Enable HoundCI
#### What are the relevant pivotal tracker stories?
#165054398

### CODE REVIEW

**Review by abayo-luc — APPROVED** [review]

**Review by niomwungeri-fabrice — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (puritybirir) [inline .vscode/launch.json]: Vscode files should not be commited

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  // Set the default
+  "editor.formatOnSave": true,
```
Comment (puritybirir) [inline .vscode/settings.json]: Vscode files should be ignored

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
+{
+  "root": true,
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true,
+    "jest": true
+  },
+  "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],
```
Comment (puritybirir) [inline .eslintrc.json]: In this line,

` "extends": ["eslint:recommended", "plugin:react/recommended", "airbnb"],`

change to  `"extends": [ "airbnb", "plugin:prettier/recommended", "prettier/react"],`

File: .eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2016,
+    "sourceType": "module"
+  },
+  "plugins": ["react", "prettier"],
+  "rules": {
```
Comment (puritybirir) [inline .eslintrc.json:22]: Under rules also add 

`{
        "newlines-between": "always"
      }`

**Review by puritybirir — CHANGES_REQUESTED** [review]
I would like a couple of rules added:

**Review by puritybirir — APPROVED** [review]

**Pull request merged**